### DISCUSSION TITLE
Transcribe audios using Loops

### DISCUSSION THREAD

**Discussion by pranavbhat12** [openai/whisper#1708] [Q&A]
I tried to transcribe a list of audio files using for loop but I found a surprising result that in one of the audio files it clearly spoke "We need to Buy some stocks" but the transcription came as "We need to Sell some stocks".But when I ran this file separately transcription was correct.What could be reason for such difference between individual inference or inference in loops.Is it taking previous text of the audios also into consideration? I don't think so but any help would be appreciated.

Note:After every audio transcription a time gap of 3 secs was given before moving to next audio.

**Comment by dgoryeo** [comment]
Hi @pranavbhat12 , I have the same experience. It seems that the model gets more hallucination / incorrect the more it runs. It would be nice if it was possible to reset the model for every run, but I have not been able to find a way to do it.
